opencti-platform/opencti-front/src/private/components/common/stix_domain_objects/StixDomainObjectContent.jsx

```diff
@@ -362,9 +362,20 @@ class StixDomainObjectContentComponent extends Component {
           ])),
         R.fromPairs,
       )(ret.images);
+
+      let pdfElementMaxWidth = 0;
+      // We need to get the children width in order to know which mode we should save the PDF
+      const elementCkEditor = document.querySelector('.ck-content.ck-editor__editable.ck-editor__editable_inline');
```

Comment:
We could not find a better way to determine the width of the element. `CKEditor` did not provide the width of the element, and neither did `htmlToPdfmake`.  Moreover, we used a querySelector to retrieve the content inside CKEditor and checked the children's width to determine the maxWidth. Depending on the result, we choose between landscape or portrait for the PDF printing